db/migrate/20161003204325_add_user_to_katello_subscription_facets.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserToKatelloSubscriptionFacets < ActiveRecord::Migration
+  def change
```

Comment:
It's probably better to have a `def up` with this and `def down` that does `remove_column`, `remove_index` & `remove_foreign_key`